project_description/__manifest__.py

```diff
@@ -1,15 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Project Description",
     "summary": "Add a description to projects",
-    "version": "10.0.1.0.0",
-    "category": "Project Management",
-    "website": "https://www.tecnativa.com",
+    "version": "11.0.1.0.0",
+    "category": "Project",
+    "website": "https://github.com/OCA/project",
     "author": "Antiun Ingeniería S.L., "
```

Comment:
Please remove this old reference, as the new company is Tecnativa

Reply:
You mean Antiun Ingeniería (http://antiun.com/) is no longer exist and it is now Tecnativa?
If so, should I change it also in models/__init__.py and models/project.py?

Also, should I change the contributors in the README.rst to:
* Tecnativa (https://www.tecnativa.com): Rafael Blasco <rafael.blasco@tecnativa.com>
* Tecnativa (https://www.tecnativa.com): Jairo Llopis <jairo.llopis@tecnativa.com>

Reply:
Well, Antiun still exists, but Antiun's Odoo division is now part of Tecnativa.

Reply:
so?

Reply:
Remove it and leave Tecnativa, please

Reply:
ok